README.md

```diff
@@ -26,7 +27,45 @@ None
 | `monit_conf_httpd_port` | bind port of http interface | `2812` |
 | `monit_conf_httpd_allow` | ACL to allow | `["127.0.0.1"]` |
 | `monit_conf_logfile_syslog_facility` | syslog facility | `log_daemon` |
-| `monit_rc` | dict of `monit` configuration files (see the example playbook) | `{}` |
+| `monit_rc` | dict of `monit` configuration fragment files (see below) | `{}` |
+| `monit_config` | string of `monit_conf_file` (see below) | `""` |
+
+## `monit_conf_include_dir`
```

Comment:
* typo: s/monit_conf_include_dir/monit_conf_extra_include_dir/ ?
* this variable is an array and the name should be plural form to clarify that it is not a single value.
* i do not understand the necessity of the variable from the example. if the directory is not created by the task 
 "Create monit_conf_extra_include_dir directories" but by the other project, i would understand the necessity.

Reply:
would you elaborate the last question? i cannot decipher it.

Reply:
my understanding is that the other project creates files to be included by monit in the directories in `monit_conf_extra_include_dir`. the directories should be created by the project which creates
the files, not by the task "Create monit_conf_extra_include_dir directories"